### PR TITLE
Explained random filenames

### PR DESCRIPTION
**Pull request by wdmearns**
Explanatory sentence appended to step #6

### CODE REVIEW

File: aws-example/README.md [hunk 1]
```diff
@@ -142,6 +142,7 @@ Statement:
 
    Downloaded files will collect in the "objects" folder.
    Files will be in [NetCDF](http://www.unidata.ucar.edu/software/netcdf/docs/netcdf_introduction.html) format and have diagnostics
-   that correspond to the arguments supplied in step 5.
+   that correspond to the arguments supplied in step 5. Note, however, that the filenames will be entirely random, i.e. details of
```
Comment (robert-lil) [inline aws-example/README.md:145]: :+1: for the wording,

It will also need to be added to the readme on the http example or on the top level readme

Comment (wdmearns) [reply aws-example/README.md:145]: OK, ta. Seems a bit specific for top-level, but have added to HTTP example

Comment (robert-lil) [reply aws-example/README.md:145]: that's fine, just squash the commits and then it'll be ready to merge

Comment (wdmearns) [reply aws-example/README.md:145]: Squashed

**Review by robert-lil — APPROVED** [review]

**Pull request merged**